Source/Core/Core/Boot/Boot_BS2Emu.cpp

```diff
@@ -214,10 +252,21 @@ bool CBoot::EmulatedBS2_GC(const DiscIO::VolumeDisc& volume,
   INFO_LOG_FMT(BOOT, "Faking GC BS2...");
 
   SetupMSR();
+  SetupHID(/*is_wii*/ false);
   SetupBAT(/*is_wii*/ false);
 
   SetupGCMemory();
 
+  // Datel titles don't initialize the postMatrices, but they have dual-texture coordinate
+  // transformation enabled. We initialize all of xfmem to 0, which results in everything using
+  // a texture coordinate of (0, 0), breaking textures. Normally the IPL will initialize the last
+  // entry to the identity matrix, but BS2 EMU skips that.
```

Comment:
This comment confuses me, doesn't this new piece of code initialize the last entry to the identity matrix?

Reply:
BS2 EMU skips running the IPL, so the initialization that the IPL performs is skipped and thus needs to be performed in BS2 EMU. But that comment can also be read as "BS2 EMU skips performing this initialization", which is misleading - I'll change it.

Reply:
I've updated the comment (I thought I did so earlier, but it seems like I didn't).